src/Bicep.Core/TypeSystem/ErrorSymbolExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Bicep.Core.SemanticModel;
+
+namespace Bicep.Core.TypeSystem
+{
+    public static class ErrorSymbolExtensions
```

Comment:
I thought we already had this somewhere...

Reply:
Actually maybe it's just git not being able to tell it's a rename?

Reply:
Yeah it's a rename. IIRC git only considers it a rename if <50% of the file is different.